modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

```diff
@@ -1,25 +1,25 @@
 ////
 Base the file name and the ID on the module title. For example:
 * file name: proc-doing-procedure-a.adoc
-* ID: [id="doing-procedure-a_{context}"]
+* ID: [id="proc-doing-procedure-a_{context}"]
```

Comment:
Why would this be included in the ID. Unlike the file name or the "module type" attribute, the ID is actually surfaced to the end-user through anchors. And the user has no interest in internal code-specific elements, such as the module type. Consider the following two URL examples:

`https://www.example.com/docs/prod-doing-stuff/`

and:

`https://www.example.com/docs/doing-stuff/`

Which one makes more sense?


```suggestion
* ID: [id="doing-procedure-a_{context}"]
```

Reply:
@rkratky this is a convention that the group agreed on before you join.

Reply:
Respectfully, it's a convention that must have been added *after* the templates were first created. I was there :)

But thanks for the explanation. I will open a separate issue.